.buildkite/pipeline.yml

```diff
@@ -69,50 +69,50 @@ steps:
-   agents:
-     queue: "docker"
- - label: test deb binary packages via docker
-   commands:
-   - eval "$SET_VERSION"
-   # Building all binary packages will take significant amount of time, so we build only one
-   # in order to ensure package generation sanity
-   - ./docker/docker-tezos-packages.sh --os ubuntu --type binary --package tezos-baker-008-PtEdo2Zk
-   - rm -rf out
-   # It takes much time to build binary package, so we do it only on master
-   branches: "master"
-   timeout_in_minutes: 90
-   agents:
-     queue: "docker"
- - label: test rpm source packages via docker
-   commands:
-   - eval "$SET_VERSION"
-   - ./docker/docker-tezos-packages.sh --os fedora --type source
-   artifact_paths:
-     - ./out/*
-   branches: "!master"
-   timeout_in_minutes: 30
-   agents:
-     queue: "docker"
- - label: test rpm binary packages via docker
-   commands:
-   - eval "$SET_VERSION"
-   # Building all binary packages will take significant amount of time, so we build only one
-   # in order to ensure package generation sanity
-   - ./docker/docker-tezos-packages.sh --os fedora --type binary --package tezos-baker-008-PtEdo2Zk
-   - rm -rf out
-   # It takes much time to build binary package, so we do it only on master
-   branches: "master"
-   timeout_in_minutes: 90
-   agents:
-     queue: "docker"
+#  - label: test deb source packages via docker
+#    commands:
+#    - eval "$SET_VERSION"
+#    - ./docker/docker-tezos-packages.sh --os ubuntu --type source
+#    artifact_paths:
+#      - ./out/*
+#    branches: "!master"
+#    timeout_in_minutes: 30
+#    agents:
+#      queue: "docker"
+#  - label: test deb binary packages via docker
+#    commands:
+#    - eval "$SET_VERSION"
+#    # Building all binary packages will take significant amount of time, so we build only one
+#    # in order to ensure package generation sanity
+#    - ./docker/docker-tezos-packages.sh --os ubuntu --type binary --package tezos-baker-008-PtEdo2Zk
+#    - rm -rf out
+#    # It takes much time to build binary package, so we do it only on master
+#    branches: "master"
+#    timeout_in_minutes: 90
+#    agents:
+#      queue: "docker"
+#  - label: test rpm source packages via docker
+#    commands:
+#    - eval "$SET_VERSION"
+#    - ./docker/docker-tezos-packages.sh --os fedora --type source
+#    artifact_paths:
+#      - ./out/*
+#    branches: "!master"
+#    timeout_in_minutes: 30
+#    agents:
+#      queue: "docker"
+#  - label: test rpm binary packages via docker
+#    commands:
+#    - eval "$SET_VERSION"
+#    # Building all binary packages will take significant amount of time, so we build only one
+#    # in order to ensure package generation sanity
+#    - ./docker/docker-tezos-packages.sh --os fedora --type binary --package tezos-baker-008-PtEdo2Zk
+#    - rm -rf out
+#    # It takes much time to build binary package, so we do it only on master
+#    branches: "master"
+#    timeout_in_minutes: 90
+#    agents:
+#      queue: "docker"
```

Comment:
Don't forget about this :P

Reply:
This is going to be uncommented only once Tezos will release v9.0 opam packages